src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discont36 = 20;
+  const discont7 = 50;
+  const rent = 40;
+  let result = 0;
+
+  if (days >= 7) {
+    result = rent * days - discont7;
```

Comment:
Just return the value everywhere
```suggestion
    return rent * days - discont7;
```